todo_app/templates/index.html

```diff
@@ -9,20 +9,34 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        {% for item in items %}
-        <li> {{item.title}} </li>
+        {% for list in lists%}
+        <h3>{{ list['name']}}</h3>
```

Comment:
You can use dot notation here

```suggestion
        <h3>{{ list.name }}</h3>
```